### PR TITLE
[ios] Introduce scale bar color settings

### PR DESCRIPTION
**Pull request by Teffan**
This pull request introduces options to change the coloring of scalebar blocks and the distance label text color.

Simulator Screen Shot - iPhone 8 - 2020-03-03 at 16 34 07

### CODE REVIEW

**Comment by julianrex** [comment]
Thanks for the contribution! We'll take a look

**Comment by arnekaiser** [comment]
Any updates on this? We'd really like to use this feature. :)

File: platform/ios/src/MGLMapView.h [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #import <UIKit/UIKit.h>
 
 #import "MGLCompassButton.h"
+#import "MGLScaleBar.h"
```
Comment (fabian-guerra) [inline platform/ios/src/MGLMapView.h:4]: Please replace this line for:

```
@class MGLScaleBar;
```

File: platform/ios/src/MGLScaleBar.h [hunk 2]
```diff
@@ -1,9 +1,29 @@
 #import <UIKit/UIKit.h>
-#import <CoreLocation/CoreLocation.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
+
+/**
+A  view that displays a scale depending of the current zoom level and cunit of length.
```
Comment (jmkiley) [inline platform/ios/src/MGLScaleBar.h:7]: nit: `cunit` -> `unit`.

**Comment by arnekaiser** [comment]
I created a new pull request with the requested changes, see #425.

**Pull request closed without merging**